controller/tests/array_action/xiv/array_mediator_xiv_tests.py

```diff
@@ -263,7 +263,7 @@ def test_create_snapshot_succeeds(self):
         xcli_snapshot = self._get_single_snapshot_result_mock(snapshot_name, snapshot_volume_name,
                                                               snapshot_capacity=size_in_blocks_string)
         self.mediator.client.cmd.snapshot_create.return_value = xcli_snapshot
-        res = self.mediator.create_snapshot(snapshot_name, snapshot_volume_name)
+        res = self.mediator.create_snapshot(snapshot_name, snapshot_volume_name, "pool1")
```

Comment:
consider using `pool_name = "pool1"`

Reply:
do you mean:
res = self.mediator.create_snapshot(snapshot_name, snapshot_volume_name, **pool="pool1"**) ?

Reply:
no, I mean use a variable, like the other arguments

Reply:
done